ui/pages/notification-details/notification-details.tsx

```diff
@@ -44,7 +44,7 @@ export default function NotificationDetails() {
   );
 
   useEffect(() => {
-    if (!id || !notificationData) {
+    if (!id) {
       redirectToNotifications();
```

Comment:
Short note (we can do this in future). Instead of ID and loading the notification modal, could we use existing notification data (or pass down the notification through props?)

Maybe this is fine, since we have a selector that we can use to get the notification otherwise fetch it.